src/resources/views/pages/mainCalendar.blade.php

```diff
@@ -0,0 +1,2072 @@
+                                            </div>
+                                            <div class="media-body">
+                                                <div class="font-w600">New sale! + $119</div>
+                                                <small class="text-muted">45 hours ago</small>
+                                            </div>
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a class="text-dark media py-2" href="javascript:void(0)">
+                                            <div class="mr-3 ml-2">
+                                                <i class="fa fa-fw fa-circle text-success"></i>
+                                            </div>
+                                            <div class="media-body">
+                                                <div class="font-w600">New sale! + $499</div>
+                                                <small class="text-muted">46 hours ago</small>
+                                            </div>
+                                        </a>
+                                    </li>
+                                </ul>
+
+                                <!-- More -->
+                                <div class="text-center">
+                                    <a class="btn btn-sm btn-light" href="javascript:void(0)">
+                                        <i class="fa fa-arrow-down mr-1"></i> Load More..
+                                    </a>
+                                </div>
+                                <!-- END More -->
+                            </div>
+                            <!-- END Yesterday -->
+                        </div>
+                    </div>
+                    <!-- END Sales Tab -->
+                </div>
+            </div>
+            <!-- END Side Overlay Tabs -->
+        </div>
+        <!-- END Side Content -->
+    </aside>
+    <!-- END Side Overlay -->
+
+    <!-- Sidebar -->
+    <!--
+        Sidebar Mini Mode - Display Helper classes
+
+        Adding 'smini-hide' class to an element will make it invisible (opacity: 0) when the sidebar is in mini mode
+        Adding 'smini-show' class to an element will make it visible (opacity: 1) when the sidebar is in mini mode
+            If you would like to disable the transition animation, make sure to also add the 'no-transition' class to your element
+
+        Adding 'smini-hidden' to an element will hide it when the sidebar is in mini mode
+        Adding 'smini-visible' to an element will show it (display: inline-block) only when the sidebar is in mini mode
+        Adding 'smini-visible-block' to an element will show it (display: block) only when the sidebar is in mini mode
+    -->
+    <nav id="sidebar" aria-label="Main Navigation">
```

Comment:
extraemos la navegación a un componente de laravel para poder reutilizarlo?
https://laravel.com/docs/8.x/blade#components
https://dev.to/ericchapman/laravel-blade-components-5c9c